webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { LoadingState } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { Table } from '../common/table';
+import { STATUS } from '../../constants';
+import MessageBox from '../common/MessageBox';
+import { translate as __ } from '../../common/I18n';
+import createModelsTableSchema from './ModelsTableSchema';
+import { getURIQuery } from '../../common/helpers';
+import Pagination from '../Pagination/Pagination';
+
+class ModelsTable extends React.Component {
+  componentDidMount() {
+    this.props.getTableItems('models', getURIQuery(window.location.href));
+  }
+
+  render() {
+    const {
+      getTableItems,
+      sortBy,
+      sortOrder,
+      error,
+      status,
+      results,
+      data: { pagination },
```

Comment:
as mentioned above, if we use the component wrapper, we won't need the data object here.